### ISSUE TITLE
Need a tag for <sectioninfo> within <reference>

### ISSUE THREAD

**Issue by hlflanagan** [rfc-format/draft-iab-xml2rfc-v3-bis#79]
We need is a change in vocabulary that says something like <sectionInfo> with possible values of section, chapter, figure, etc, that we can include in a reference. Currently have to use <seriesInfo> and that seems inappropriate. 

Ideally, this would look like:
<reference anchor='Unicode' target='http://www.unicode.org/versions/Unicode12.1.0/'>
 <front>
  <title>The Unicode Standard</title>
  <author>
   <organization>The Unicode Consortium</organization>
  </author>
  <date month="May" year="2019" day="7" />
 </front>
 <sectionInfo name='Section' value='3.2' />
 <seriesInfo name='Unicode' value='12.1.0' />
</reference>

Which would result in:
[Unicode]  The Unicode Consortium, "The Unicode Standard",
              Section 3.2, Unicode 12.1.0, May 2019,
              <http://www.unicode.org/versions/Unicode12.1.0/>.

**Comment by reschke** [comment]
So you want to have a reference to a specific part of a document, and that should be in the reference, not in the prose making the reference?

Why not say "see Section 3.2 of [UNICODE]"?

**Comment by hlflanagan** [comment]
Particularly in the case of very large documents (such as the Unicode spec) it is helpful to the reader to have multiple references (same document, different sections).

**Comment by reschke** [comment]
Well, you could put "Section 3.2" into \<refcontent>, no?

See <https://greenbytes.de/tech/webdav/rfc7991.html#element.refcontent>

**Comment by hlflanagan** [comment]
True, that could definitely serve the purpose. <sectionInfo> could be more specific in terms of format, <refcontent> for miscellaneous cases that don't fit into a better model. Alternatively, if the description of <refcontent> is updated, that could be a compromise.

**Comment by paulehoffman** [comment]
I'm with Julian on this one. Having different reference elements for the same document will cause some people to wonder if they need a different reference for each section of a shorter document like an RFC.

**Comment by hlflanagan** [comment]
It would be noted in the style guide as something for large documents, not as the usual case.
